ckanext/recombinant/views.py

```diff
@@ -598,14 +608,18 @@ def _process_upload_file(lc, dataset, upload_file, geno, dry_run):
                 pgerror = e.error_dict['records'][0]
             if isinstance(pgerror, dict):
                 pgerror = u'; '.join(
-                    k + u': ' + u', '.join(v)
+                    k + u': ' + u', '.join(_(e.split('\uF8FF')[0]).format(e.split('\uF8FF')[1]) if '\uF8FF' in e else _(e) for e in v)
```

Comment:
This is repeated a few times, how about a function with a good name to reduce duplication and indicate what all the `\uF8FF` is about